ngraph/python/tests/test_ngraph/test_hswish.py

```diff
@@ -0,0 +1,29 @@
+# ******************************************************************************
```

Comment:
Perhaps this test could be moved to a file with all other activation functions?

Reply:
I couldn't find such a file, do you know the path?

Reply:
I moved the test to `ngraph/python/tests/test_ngraph/test_ops_unary.py`